ChemistryLib/CreatePhreeqcInterface.cpp

```diff
@@ -0,0 +1,86 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <iosfwd>
```

Comment:
I'd think you can drop this include here.

Reply:
Good catch. Discarded.